ambry-cloud/src/main/java/com.github.ambry.cloud/azure/AzureMetrics.java

```diff
@@ -41,13 +40,14 @@
   public static final String BLOB_DELETION_TIME = "BlobDeletionTime";
   public static final String BLOB_DELETE_ERROR_COUNT = "BlobDeleteErrorCount";
   public static final String CONFIG_ERROR_COUNT = "ConfigErrorCount";
-  public static final String BLOB_UPLOAD_RATE = "BlobUploadRate";
+  public static final String BACKUP_SUCCESS_LATENCY = "backupSuccessLatency";
+  public static final String BACKUP_SUCCESS_BYTE_RATE = "backupSuccessByteRate";
+  public static final String BACKUP_ERROR_COUNT = "backupErrorCount";
```

Comment:
I don't like this change.  BlobUploadErrorCount was more consistent with the other upload counts.
Also, your metric names are camel case inconsistent with existing ones.

Reply:
The original BlobUploadErrorCount includes both azure blob upload failure and document upsert failure. The name is confusing. 
Since It's  blob upload error + document error, I rename it to BackupErrorCount to be clear. 

Also, the change moved blob upload related metric inside `uploadIfNotExists`. (blob upload error count can be derived from `blobUploadRequestCount` - `blobUploadSuccessCount`)


Will change the constant string style.

Reply:
Besides the change, Looks like `updateErrorMetrics` is over reused. It was called two place, create and update. We can't know what caused error directly.

Reply:
I think updateErrorMetrics is okay, it distinguishes between Cosmos and storage errors.
I'm fine with the other change.